_includes/v21.2/known-limitations/restore-multiregion-match.md

```diff
@@ -0,0 +1,50 @@
+[`REGIONAL BY TABLE`](multiregion-overview.html#regional-tables) tables can be restored **only** if the [table locality](multiregion-overview.html#table-locality) matches that of the target database.
+
+    The following must also be true for `RESTORE` to be successful:
+
+    * The regions of the source database and the regions of the destination database have the same set of regions.
+    * The regions were added to each of the databases in the same order.
+    * The databases have the same primary region.
+
+    The following example would be considered as having **mismatched** regions because the database regions were not added in the same order and the primary regions do not match.
+
+    Running on the source database:
+
+    ~~~ sql
+    > ALTER DATABASE source_database SET PRIMARY REGION "us-east1";
+    ~~~
+    ~~~ sql
+    > ALTER DATABASE source_database ADD region "us-west1";  
+    ~~~
+
+    Running on the destination database:
+
+    ~~~ sql
+    > ALTER DATABASE destination_database SET PRIMARY REGION "us-west1";
+    ~~~
+    ~~~ sql
+    > ALTER DATABASE destination_database ADD region "us-east1";  
+    ~~~
+
+    Furthermore, this scenario has mismatched regions between the databases since the regions were not added to the database in the same order.
```

Comment:
I would find it clearer if this said something like "In addition, the following scenario ..." for the following reasons:
1. "furthermore" makes it sound like this is a continuation of the previous example
2. "this" is somewhat ambiguous because it makes me think it could be referring to the previous example